browser/src/Editor/NeovimEditor.tsx

```diff
@@ -283,6 +283,30 @@ export class NeovimEditor implements IEditor {
             }
         })
 
+        const normalizePath = (fileName: string) => fileName.split("\\").join("/")
+
+        ipcRenderer.on("open-files", (_evt: any, message: string, files: string[]) => {
+
+            // Open the first file.
+            // If the current buffer is named, then open in a new tab or split.
+            // If the current buffer has modifications pending, then open in a new tab or split.
+            // If the current buffer had no modification and is a new file, open in the current buffer.
+
+            this._neovimInstance.command("if bufname('%') != ''\n" +
```

Comment:
Makes sense, I was wondering if there was an easier place to put something like this, since it doesn't read very nicely how I had it.